tools/status/report.sh

```diff
@@ -10,24 +13,53 @@ RUN_ID=$3
 TEST_OUTCOME=$4
 QA_CHECKS_OUTCOME=$5
 
+# Ensure connector is prefixed with connectors/
+# TODO (ben): In the future we should just hard error if this is not the case
+if [[ $CONNECTOR != *"/"* ]]; then
```

Comment:
👍 I don't think a "/" would be in a connector name

Reply:
haha ya, i didn't feel the best about this but bash wise I think its good